frontend/app/component/form/data_entry/candidates_votes/CandidatesVotesForm.tsx

```diff
@@ -32,7 +32,7 @@ export interface CandidatesVotesFormProps {
 
 export function CandidatesVotesForm({ group }: CandidatesVotesFormProps) {
   const formRef = React.useRef<CandidatesVotesFormElement>(null);
-
+  const ignoreWarnignsRef = React.useRef<HTMLInputElement>(null);
```

Comment:
`ignoreWarningsRef` is consistently misspelled. I suggest we change it to `acceptWarningsRef` instead, to reflect what we call this checkbox in our use cases.